package.json

```diff
@@ -8,6 +8,8 @@
     "@fortawesome/free-regular-svg-icons": "^5.15.4",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.1.15",
+    "@terra-ui-packages/components": "*",
+    "@terra-ui-packages/core-utils": "*",
```

Comment:
The Terra UI app depends on the latest code in each package. Yarn resolves imports from these packages to the files in the packages directory.